scripts/update-framework-results.py

```diff
@@ -23,21 +23,23 @@
 
 from dmscripts.env import get_api_endpoint_from_stage
 from dmscripts.logging import configure_logger
-from dmscripts.insert_dos_framework_results import insert_result
```

Comment:
Turns out there is a oneoff script using it so I haven't killed it, but have refactored things a bit more instead.